titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/validator/FailJobValidator.java

```diff
@@ -46,4 +56,9 @@
     public Mono<JobDescriptor> sanitize(JobDescriptor entity) {
         return Mono.error(TitusServiceException.invalidArgument(ERR_DESCRIPTION));
     }
+
+    @Override
+    public ValidationError.Type getErrorType() {
```

Comment:
This implementation is identical in all cases, and could be moved to an abstract class. This will also make it easier to add new methods in the future if needed.